metricbeat/module/docker/healthcheck/_meta/fields.yml

```diff
@@ -2,6 +2,8 @@
   type: group
   description: >
     Docker container metrics.
```

Comment:
This seems to be off. Not introduced in this PR but I wonder if we should change it to `Docker healthcheck metrics.` 

Should we add an empty line below or do you want to have it all in one paragraph?

Reply:
Docker healthcheck metrics sounds better and more specific to describe it.
Will adjust.